app.py

```diff
@@ -1,7 +1,9 @@
-from app import manager
 from flask import Flask, request
+from injector import Injector
+from app.manager import Manager
 
 app = Flask(__name__)
+manager = Injector().get(Manager)
 
 
 @app.route('/init', methods=['GET', 'POST'])
```

Comment:
It should be only POST, but since it's a training and the front-end is not part of it, it's much easier to debug test it with GET by the browser, so I added this method too.